src/components/Map/mocks.js

```diff
@@ -0,0 +1,119 @@
+import {rest} from 'msw';
+
+import {BASE_URL} from 'api-mocks';
+
+export const mockPdokApiResponse = rest.get(`${BASE_URL}formio/mapsearch`, (req, res, ctx) => {
```

Comment:
```suggestion
export const mockPdokApiResponseGet = rest.get(`${BASE_URL}formio/mapsearch`, (req, res, ctx) => {
```
please follow the convention of the mock names